### PR TITLE
llama dupe feature

### PR DESCRIPTION
**Pull request by whoImT**

### CODE REVIEW

**Review by FracturedCode — APPROVED** [review]
noice. I'll merge this into a feature branch for modification

File: src/main/java/net/alephsmp/aleph_carpet/mixins/llamaDupe/PlayerManagerMixin.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+@Mixin(PlayerManager.class)
+public abstract class PlayerManagerMixin{
+
+    @Final
+    @Shadow private static final Logger LOGGER = LogManager.getLogger();
+
+    @Redirect(method = "remove", at = @At(value = "INVOKE", target = "Lnet/minecraft/server/network/ServerPlayerEntity;hasVehicle()Z"))
+    private boolean llamaDupe(ServerPlayerEntity serverPlayerEntity){
+        if (serverPlayerEntity.hasVehicle()) {
```
[the remark this answers is not in the document]
Comment (FracturedCode) [reply src/main/java/net/alephsmp/aleph_carpet/mixins/llamaDupe/PlayerManagerMixin.java:21]: image
just to make sure I understand what is going on, we are trying to make this piece of code always run whenever the feature is on? Or is there another component to it

[the remark this answers is not in the document]
Comment (whoImT) [reply src/main/java/net/alephsmp/aleph_carpet/mixins/llamaDupe/PlayerManagerMixin.java:21]: Ye, but we set the removed value to false if

[the remark this answers is not in the document]
Comment (whoImT) [reply src/main/java/net/alephsmp/aleph_carpet/mixins/llamaDupe/PlayerManagerMixin.java:21]: We should probably remove the redirect and just modify the variables

[the remark this answers is not in the document]
Comment (FracturedCode) [reply src/main/java/net/alephsmp/aleph_carpet/mixins/llamaDupe/PlayerManagerMixin.java:21]: ah, I see

**Pull request merged**